internal/target/incus.go

```diff
@@ -397,6 +397,9 @@ func (t *InternalIncusTarget) CreateNewVM(apiDef incusAPI.InstancesPost, storage
 		"pool":          storagePool,
 		"source":        bootISOImage,
 		"boot.priority": "10",
+		"readonly":      "true",
+		"io.bus":        "virtio-blk",
+		"io.cache":      "writeback",
```

Comment:
Do we still need the io.cache bit now that we've fixed the source of the corruption problem?

Reply:
I'll run a quick test to be sure. I'm guessing probably not though.

Reply:
Yup, seems to work without it. Removed it now.